light_curves/light_curve_classifier.md

```diff
@@ -8,11 +8,11 @@ jupytext:
 kernelspec:
   display_name: science_demo
   language: python
-  name: conda-env-science_demo-py
+  name: python3
```

Comment:
With this change, I have to manually select the kernel. Changing this back to `name: conda-env-science_demo-py` works out of the box.